src/app/pages/icon-template/icon-template.page.html

```diff
@@ -0,0 +1,25 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button defaultHref="/home"></ion-back-button>
+    </ion-buttons>
+    <ion-title>Templates</ion-title>
+  </ion-toolbar>
+</ion-header>
+<ion-content>
+  <ion-item lines="full">
+    <ion-label class="ion-text-wrap">
+      Setting icon template.
+    </ion-label>
+  </ion-item>
+  <ion-item>
+    <ion-label>Port</ion-label>
+    <ionic-selectable [(ngModel)]="port" itemValueField="id"
+      itemTextField="name" [items]="ports" [canSearch]="true"
+      (onChange)="portChange($event)">
+      <ng-template ionicSelectableIconTemplate>
+        <ion-icon name="beer"></ion-icon>
```

Comment:
Beer :) that's funny, but let's change to something more appropriate for a list of ports, or on the other hand we can keep the beer icon, but then need to change the list to display sorts of beer :) Up to you